lib/screens/authentication/widgets/verify_button.dart

```diff
@@ -1,14 +1,218 @@
 import 'package:flutter/material.dart';
+import 'package:saasify/screens/onboarding/companies_screen.dart';
 
+import '../../../data/customer_cache/customer_cache.dart';
+import '../../../data/models/authentication/authentication_model.dart';
+import '../../../di/app_module.dart';
 import '../../../utils/constants/string_constants.dart';
 import '../../../widgets/primary_button.dart';
+import '../../../widgets/sidebar.dart';
 
 class AuthVerifyButton extends StatelessWidget {
-  const AuthVerifyButton({super.key});
+  final CustomerCache _customerCache = getIt<CustomerCache>();
+
+  AuthVerifyButton({super.key});
 
   @override
   Widget build(BuildContext context) {
     return PrimaryButton(
-        onPressed: () {}, buttonTitle: StringConstants.kVerify);
+        onPressed: () {
+          _customerCache.setIsLoggedIn(true);
+          _customerCache.setUserId("OvcqwTKNuHSL5ILAOZ6Y3OGcmrj2");
+          _customerCache.setCompanyId('1702472309');
+          _customerCache.setBranchId(1);
+          _customerCache.setUserContact(918888881800);
+          _customerCache.setUserName("Adityaaaaaa");
+          SideBar.userContact = 918888881800;
+          SideBar.userName = 'Adityaaaaaa';
```

Comment:
Please remove hardcoded values, everything should be dynamic

Reply:
yes all this is there in authentication bloc but to see the ui i had to hardcode it.